src/public/sass/UI/product/_modal-form.scss

```
@@ -0,0 +1,128 @@
+@use 'base/variables' as v;
+@use 'base/mixins' as m;
+
+.modal-form {
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  align-items: flex-end;
+  margin: 0 auto;
+
+  &__title {
+    margin: 0 auto;
+    margin-bottom: 2rem;
+    color: v.$blue;
+    font-size: 1.5rem;
+    line-height: 1;
+    white-space: nowrap;
+
+    @include m.phoneXL {
+      font-size: 1.9rem;
+    }
+    &--big {
+      display: block;
+      font-size: 2.5rem;
+      font-weight: bold;
+
+      @include m.phoneXL {
+        font-size: 3rem;
+      }
+    }
+  }
+  &__field {
+    display: flex;
+    width: 100%;
+    flex-direction: column;
+  }
+  &__label {
+    margin-bottom: 0.5rem;
+    color: v.$teal-blue;
+    font-size: 1.8rem;
+    font-weight: bold;
+  }
+  &__input,
+  &__textarea {
+    padding: 1rem;
+    border: 0.1rem solid v.$teal-blue;
+    border-radius: 1rem;
+    margin-bottom: 2rem;
+    background-color: v.$white;
+    font-size: 1.6rem;
+    resize: none;
+    transition: border-color 300ms ease;
+
+    &:focus {
+      border-color: v.$sky-blue;
+      outline: none;
+    }
+    &::placeholder {
+      color: v.$gray;
+    }
+    &--product {
+      padding: 0.8rem 1rem;
+      color: v.$blue;
+      cursor: not-allowed;
+      font-weight: bold;
+      text-align: center;
+
+      @include m.phoneXL {
+        font-size: 2rem;
+      }
+    }
+  }
+  &__submit {
+    padding: 0.5rem 3.5rem;
+    border: none;
+    border-radius: 10rem;
+    background-color: v.$teal-blue;
+    color: v.$white;
+    font-size: 2rem;
+    font-style: italic;
+    font-weight: bold;
+    text-transform: capitalize;
+    transition: background-color 300ms ease;
+
+    &:hover {
+      cursor: pointer;
+    }
+  }
+}
+.custom-modal-form {
+  -webkit-backdrop-filter: blur(3px);
+  backdrop-filter: blur(3px);
+  background-color: rgb(1 1 1 / 0.3);
+  cursor: auto;
+
+  .tingle-modal {
+    &__close {
+      visibility: hidden;
+
+      @media (max-width: 540px) {
+        background-color: v.$teal-blue;
+        visibility: visible;
+      }
+    }
+  }
+  .tingle-modal-box {
+    width: 100%;
+    max-width: 50rem;
+    border: 0.1rem solid v.$teal-blue;
+    border-radius: 2.5rem;
+    -webkit-box-shadow: 0px 0px 25px 8px rgba(0, 0, 0, 0.51);
+    -moz-box-shadow: 0px 0px 25px 8px rgba(0, 0, 0, 0.51);
+    box-shadow: 0px 0px 25px 8px rgba(0, 0, 0, 0.51);
```

```suggestion
    box-shadow: 5px 5px 12px 5px #00000082;
```
Again prefixes not needed and please change the box shadow with this one.